crates/compiler/fmt/src/annotation.rs

```diff
@@ -492,7 +494,12 @@ fn fmt_tag_collection<'a>(
 }
 
 impl<'a> Nodify<'a> for Tag<'a> {
-    fn to_node<'b>(&'a self, arena: &'b Bump, parens: Parens) -> Spaces<'b, Node<'b>>
+    fn to_node<'b>(
+        &'a self,
+        arena: &'b Bump,
+        parens: Parens,
+        flags: MigrationFlags,
```

Comment:
Not actionable rn but we should wrap at least arena and flags in a Ctx struct so all call sites don't need to be updated in the future.

Reply:
Yeah, that change will be large itself